mParticle-Kochava/MPKitKochava.m

```diff
@@ -272,6 +286,12 @@ - (MPKitExecStatus *)continueUserActivity:(NSUserActivity *)userActivity restora
     return [[MPKitExecStatus alloc] initWithSDKCode:@(MPKitInstanceKochava) returnCode:MPKitReturnCodeSuccess];
 }
 
+- (MPKitExecStatus *)setATTStatus:(MPATTAuthorizationStatus)status withATTStatusTimestampMillis:(NSNumber *)attStatusTimestampMillis  API_AVAILABLE(ios(14)){
+    KVATracker.shared.appTrackingTransparency.enabledBool = YES;
```

Comment:
should this be gated by a check on the ATT status value? E.g. only run this if the status is accepted or similar?

Reply:
No, it should done either way. This flag is only to note if ATT is being used at all.